clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1266,9 +1271,9 @@ def viscode_to_session(viscode):
         M00 if is the baseline session or the original session name capitalized
     """
     if viscode == "bl" or viscode == "m0":
-        return "ses-M00"
+        return "ses-M000"
```

Comment:
You need to update the docstring because the function does something different than what is documented now.
Some tests would be great !